frontend-api/build.gradle

```diff
@@ -14,7 +14,8 @@ dependencies {
             configurations.nimbus,
             configurations.sqs,
             configurations.bouncycastle,
-            project(":shared")
+            project(":shared"),
+            "software.amazon.awssdk:cloudwatch:2.17.128"
```

Comment:
This could be managed at the top level build.gradle and brought in similar to others. It also shares a SDK with SQS so that version number could be put in a dependency version so there's a single place where it is upgraded.